lib/extensions/install.py

```diff
@@ -4,18 +4,48 @@
 # Licensed under the GNU GPL version 3.0 or later.  See the file LICENSE for details.
 
 from .base import InkstitchExtension
-from ..api import APIServer
-from ..gui import open_url
+import os
+import sys
+from glob import glob
+from ..utils import get_bundled_dir, guess_inkscape_config_path
+from inkex import errormsg
+from ..i18n import _
 
 
 class Install(InkstitchExtension):
-    def __init__(self):
-        InkstitchExtension.__init__(self)
+    def __init__(self, *args, **kwargs):
+        InkstitchExtension.__init__(self, *args, **kwargs)
 
     def effect(self):
-        api_server = APIServer(self)
-        port = api_server.start_server()
-        electron = open_url("/install?port=%d" % port)
-        electron.wait()
-        api_server.stop()
-        api_server.join()
+        path = os.path.join(guess_inkscape_config_path(), 'palettes')
```

Comment:
It is very unlikely that palette path is guessed wrong since inkex helps us in the guessing game. So for this extension there is no real a need to use electron and this is part of moving away from it.

Reply:
Perfect, thank you new inkex!